thirdeye/thirdeye-frontend/app/api/self-serve.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * GET request for all alert subscription groups
+ * @see {@link namepathOrURL|link text}
```

Comment:
This is a great idea Steve. Thanks for doing it! However, it all `const` or static definitions. Can we find a more formal folder where we keep all global consts? like say `app/utils/api/self-serve.js`
i think i saw `utils/constant.js` there too. SO maybe good to group all kind of constant or static helper files there? Or where we keep `common` files of this type.

Reply:
Yes, I like that as a location for it.